scripts/image-builder/configure.sh

```diff
@@ -6,15 +6,15 @@ OSVERSION=$(awk -F: '{print $5}' /etc/system-release-cpe)
 sudo dnf install -y git osbuild-composer composer-cli ostree rpm-ostree \
     cockpit-composer cockpit-machines bash-completion podman runc genisoimage \
     createrepo yum-utils selinux-policy-devel jq wget lorax rpm-build \
-    containernetworking-plugins
+    containernetworking-plugins caddy
```

Comment:
`caddy` is in the EPEL, it should be move to L17

Reply:
Thanks, fixed